src/components/recordingcreator/recordingfields.template.html

```diff
@@ -1,20 +1,8 @@
 <div class="convertRecordingsContainer hide infoBanner" style="margin: 1em 0 2em;">
```

Comment:
This seems Premiere related.

Reply:
I didn't remove this section because I honestly have no clue how well this section was scrubbed of premiere elements, I don't have a tuner for live TV so I couldn't test.

Reply:
But in `recordingfields.js`, you're only ever adding the `hide` css class and it's already the default.